sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/ServiceBusReceiverAsyncClientIntegrationTest.java

```diff
@@ -586,6 +518,17 @@ void peekMessagesFromSequence(MessagingEntityType entityType) {
         StepVerifier.create(receiver.peekMessagesAt(maxMessages, fromSequenceNumber))
             .expectNextCount(maxMessages)
             .verifyComplete();
+
+        // cleanup
+        StepVerifier.create(receiver.receiveMessages())
```

Comment:
Same with this... wouldn't we expect `receiver.receiveMessages().take(2)` then not cancel?